test.js

```diff
@@ -42,6 +44,24 @@ test('output beyond terminal height', t => {
 	t.is(terminal.state.getLine(2).str, '');
 });
 
+test('output beyond terminal height with color', t => {
+	const {terminal, log} = setup({rows: 3, columns: 20});
+
+	log('\u001B[32m√\u001B[39mline 1\nline 2\nline 3');
+	t.is(terminal.state.getLine(0).str, 'line 2');
+	t.is(terminal.state.getLine(1).str, 'line 3');
+	t.is(terminal.state.getLine(2).str, '');
```

Comment:
Without this fix it will output like this.
```js
	t.is(terminal.state.getLine(0).str, 'e 3');
	t.is(terminal.state.getLine(1).str, '');
	t.is(terminal.state.getLine(2).str, '');
```